readme.md

```diff
@@ -55,8 +55,7 @@ otherwise, a row refers to a minor version and all associated patch versions.
 
 | Ignite CLI  | Cosmos SDK  | IBC                  | Notes                                                         |
 |-------------|-------------|----------------------|---------------------------------------------------------------|
-| v0.26.2     | v0.47.2     | v7.0.0               |                                                               |
-| v0.26.1     | v0.47.1     | v7.0.0               |                                                               |
```

Comment:
Why were these releases removed?

Reply:
The matrix table needed to be corrected. I made a mistake. There are break changes. We should update a minor version, not only a patch